### ISSUE TITLE
Cannot connect to GHE instance behind private CA certificate

### ISSUE THREAD

**Issue by AaronJaegerVA** [microsoft/vscode-pull-request-github#6135]
- Extension version: 0.92.0
- VSCode Version: 1.92.0
- OS: RHEL8
- Repository Clone Configuration (single repository/fork of an upstream repository): Single Repo
- Github Product (Github.com/Github Enterprise version x.x.x): GHE 3.11.11

Steps to Reproduce:

1. Attempt to connect to GHE instance
2. Observe output from extension

> 2024-08-07 12:58:22.950 [info] Trying to use globalState for assignableUsers.
> 2024-08-07 12:58:22.955 [warning] GitHubServer> No response from host https://github.ec.example.com/org/repo: request to https://github.ec.example.com/api/v3/rate_limit failed, reason: unable to get local issuer certificate

**Comment by alexr00** [comment]
@AaronJaegerVA is this still happening?

**Comment by AaronJaegerVA** [comment]
They recently upgraded the GHE instance. Let me try again to make sure something wasn't fixed via the upgrade.

**Comment by AaronJaegerVA** [comment]
@alexr00,

Here is what I get when I attempt to Sign In to the GFE via the GH Pull Request extension.

> 2024-08-14 19:55:54.544 [warning] /home/user/.ssh/config: ENOENT: no such file or directory, open '/home/user/.ssh/config'
> 2024-08-14 19:55:54.544 [info] Activation> Extension version: 0.92.0
> 2024-08-14 19:55:56.556 [info] Looking for git repository
> 2024-08-14 19:55:56.556 [info] Found 0 repositories during activation
> 2024-08-14 19:55:56.556 [info] Git repository found, initializing review manager and pr tree view.
> 2024-08-14 19:55:56.560 [info] Registering git provider
> 2024-08-14 19:55:56.560 [info] Review+0> Validate state in progress
> 2024-08-14 19:55:56.560 [info] Review+0> Validating state...
> 2024-08-14 19:55:56.728 [info] Found GitHub remote for folder /projects/helm-valip-ocp-cluster-config
> 2024-08-14 19:55:56.729 [info] Trying to use globalState for assignableUsers.
> 2024-08-14 19:55:56.745 [info] Review+0> No matching pull request metadata found locally for current branch master
> 2024-08-14 19:55:56.745 [info] Review+0> No matching pull request metadata found on GitHub for current branch master
> 2024-08-14 19:55:56.899 [info] GlobalState does not exist for assignableUsers.
> 2024-08-14 20:07:25.344 [info] Repo state for file:///projects/helm-valip-ocp-cluster-config changed.
> 2024-08-14 20:07:25.349 [info] Repo file:///projects/helm-valip-ocp-cluster-config has already been setup.

**Comment by alexr00** [comment]
Looks like the issue is no longer occurring then.

**Comment by AaronJaegerVA** [comment]
@alexr00,

It _is_ still happening and the logs I gave above are what I see when trying to to activate the extension. I am never able to log in and get it working.

/reopen
